pkg/kubernetes/uninstall.go

```diff
@@ -6,30 +6,17 @@
 package kubernetes
 
 import (
-	"errors"
-	"fmt"
-
-	cli_ver "github.com/dapr/cli/pkg/version"
-	"github.com/dapr/cli/utils"
+	helm "helm.sh/helm/v3/pkg/action"
 )
 
-// Uninstall removes Dapr
-func Uninstall(version string) error {
-	if version == daprLatestVersion {
-		v, err := cli_ver.GetLatestRelease(cli_ver.DaprGitHubOrg, cli_ver.DaprGitHubRepo)
-		if err != nil {
-			return fmt.Errorf("cannot get the manifest file: %s", err)
-		}
-
-		version = v
-	}
-
-	var daprManifestPath string = fmt.Sprintf("https://github.com/dapr/dapr/releases/download/%s/dapr-operator.yaml", version)
-
-	_, err := utils.RunCmdAndWait("kubectl", "delete", "-f", daprManifestPath)
+// Uninstall removes Dapr from a Kubernetes cluster
+func Uninstall(namespace string) error {
```

Comment:
We need to put a note in breaking changes that, cli installs in k8s till 0.9.0 must be uninstalled with cli 0.9.0 and then get the new cli.

Reply:
Sure, planning to do that.